<?xml version="1.0"?>
<launch>
  <arg name="base_serial_port" 		default="/dev/ttyACM0"/>
  
  <arg name="base_host" 			default="10.32.114.229"/>
  <arg name="base_port"				default="16140"/>
  <arg name="rover_host" 			default="10.21.114.246"/>
  <arg name="rover_port"			default="16140"/>
	
	<group>
		<push-ros-namespace namespace = "base"/>
		<node name="f9p" pkg="ublox_read_2" exec="ublox_ros">
			<param name="serial_port"       value="$(var base_serial_port)"/>
			<param name="rover_quantity"    value="1"/>
			<param name="base_type"         value="moving"/>

			<param name="local_host"       value="$(var base_host)"/>
			<param name="local_port"        value="$(var base_port)"/>
			<param name="rover_host"        value="$(var rover_host)"/>
			<param name="rover_port"        value="$(var rover_port)"/>

			<param  from="$(find-pkg-share ublox_read_2)/params/ublox.yaml"/>
		</node> 
	</group>
	
</launch>
